addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
> you are supposed to "deprecate" for at least one release.

not possible in this case because the API (I still call it crappy) calls into the very core of the application. and this did already change, so no way to depreciate.

why don't we drop "backwards-compatibility" at all if this is considered something not strict. if it is handled not strict, the message to addon devs is: your addon may work or not. this depends on how important Kodi considers certain parts of the api. Ah, sorry, your addon was not considered important. That's the reason why it still crashes though you rely on this compatibility infio.